.evergreen/config.yml

```diff
@@ -3560,6 +3477,111 @@ tasks:
       - func: setup aws env
       - func: remove aws-credential-providers
       - func: run aws ECS auth test
+  - name: run-spec-benchmark-tests-node-server
```

Comment:
Is there a reason this was moved 2000 lines lower? I think the diff might be clearer if we keep it in the same place and then added the new changes.

Reply:
It's just a nature of where the script inserts this into the array, it not impossible to put it in the same spot but I have to splice and edit the TASKS array in place which could be brittle with future changes to the generate script that just expects all the code to keep pushing on to it.

Reply:
Looking into this it wouldn't be possible to splice them into the same place they were in unless I changed the:
```
const fileData = yaml.load(fs.readFileSync(`${__dirname}/config.in.yml`, 'utf8'));
fileData.tasks = (fileData.tasks || [])
  .concat(BASE_TASKS)
  .concat(TASKS)
```
tasks that are read in from the .in file, the reason these where higher up is because the manually written ones come first

Reply:
I see, thanks for explaining. This looks good to me then!